StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1011ClosingSquareBracketsMustBeSpacedCorrectly.cs

```diff
@@ -22,8 +21,9 @@ namespace StyleCop.Analyzers.SpacingRules
     /// line.</para>
     ///
     /// <para>A closing square bracket must be followed by whitespace, unless it is the last character on the line, it
-    /// is followed by a closing bracket or an opening parenthesis, it is followed by a comma or semicolon, or it is
-    /// followed by certain types of operator symbols.</para>
+    /// is followed by a closing bracket or an opening parenthesis, it is followed by a comma or semicolon, it is
+    /// followed by a alignment component or format string component, or it is followed by certain types of operator
```

Comment:
I would prefer to see `string interpolation alignment component or string interpolation formatting component`, instead of `alignment component or format string component`.